lib/db_helper.rb

```diff
@@ -204,7 +218,18 @@ def self.build_remap_query_parts(table, columns, skip_max_length_violations)
     end
   end
 
-  def self.log_remap_message(table, rows_updated, skipped_counts)
+  def self.skipped_transform_counts(table, query_parts, skip_max_length_violations, params)
+    return unless skip_max_length_violations && query_parts[:skipped_sums].any?
+
+    skipped = DB.query_hash(<<~SQL, params).first
+      SELECT #{query_parts[:skipped_sums].join(", ")}
+      FROM \"#{table}\"
```

Comment:
There's no need to escape the quote inside a heredoc string. `\"` could be replaced by `"` in a couple of SQL statements.